draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1147,26 +1148,29 @@ requirement above).
 
 If the log is unable to process a client's request, it MUST return an HTTP
 response code of 4xx/5xx (see [RFC7231]), and, in place of the responses
-outlined in the subsections below, the body SHOULD be a JSON structure
-containing at least the following field:
-
-error_message:
+outlined in the subsections below, the body SHOULD be a JSON Problem Details
+Object (see [RFC7807] Section 3), containing:
+
+type:
+: A URN reference identifying the problem. To facilitate automated diagnostics
+  of errors, this document defines a set of standard tokens for use in the
+  `type` field, within the URN namespace of: "urn:ietf:params:trans:error:".
+  Other than the generic codes detailed here, each error type is specific to
+  the type of request. Specific errors are defined in the respective sections
+  below.
+
+detail:
 : A human-readable string describing the error which prevented the log from
   processing the request.
 
 : In the case of a malformed request, the string SHOULD provide sufficient
   detail for the error to be rectified.
 
-error_code:
-: An error code readable by the client. Other than the generic codes detailed
-  here, each error code is specific to the type of request. Specific
-  errors are specified in the respective sections below. Error codes are fixed
-  text strings.
 
 |---------------+---------------------------------------------|
-| Error Code    | Meaning                                     |
+| Type          | Detail                                      |
```

Comment:
I think "type" and "detail" (i.e., no capitals) would be less confusing column headers for these tables.

Reply:
Done, here and throughout.